clustered/osgi-test/src/test/java/org/ehcache/osgi/ClusterSupport.java

```diff
@@ -31,8 +32,10 @@
 
   public static Cluster startServer(Path serverDirectory) throws IOException {
     Path kitLocation = Paths.get(System.getProperty("kitInstallationPath"));
-    int tsaPort = selectAvailableEphemeralPort();
-    int tsaGroupPort = selectAvailableEphemeralPort();
+
+    PortManager portManager = PortManager.getInstance();
+    PortManager.PortRef tsaPort = portManager.reservePort();
+    PortManager.PortRef tsaGroupPort = portManager.reservePort();
```

Comment:
I don't know how `PortRef` and such are implemented, but the way we use them in Galvan and Angela, we make sure to close these resources at the end to "release" the port so that it could be picked up again. Some port exhaust otherwise could happen since the file is common to several processes (from what I remember from the port manager impl we were using back then).